packages/core/src/components/sections/Hero/Overrides.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  Hero as UIHero,
+  HeroImage as UIHeroImage,
+  HeroHeader as UIHeroHeader,
+} from '@faststore/ui'
+
+import HeroCustomizations from 'src/customizations/components/overrides/Hero'
+
+const heroComponentsCustomization = {}
+
+const heroPropsCustomization = {} as any
+
+Object.entries(HeroCustomizations.components).forEach(([key, value]) => {
+  if (value.Component) {
+    heroComponentsCustomization[key] = value.Component
+  }
+})
+
+Object.entries(HeroCustomizations.components).forEach(([key, value]) => {
+  if (value.props) {
+    heroPropsCustomization[key] = value.props
+  }
+})
+
+const Components = {
+  Hero: UIHero,
```

Comment:
I think we have discussed this before, but this component is being override in more than one place 

Since homepage and PLP is using the same Hero's section 🤔 
<img width="1052" alt="image" src="https://github.com/vtex/faststore/assets/3356699/b19b35ef-5b90-4970-b1cb-25b39a63105a">

<img width="1050" alt="image" src="https://github.com/vtex/faststore/assets/3356699/0c574765-89da-4c37-a6ee-167a6d27af8b">

Reply:
When overriding a component, you are doing it globally for this section, so it's an expected behavior.